src/promptflow/tests/test_configs/flows/python_tool_with_composite_image/incomplete_inputs.jsonl

```diff
@@ -0,0 +1,2 @@
+{"image_dict":{"image_1": {"data:image/jpg;path": "logo.jpg"},"image_2": {"data:image/png;path": "logo_2.png"}}}
```

Comment:
What happens if image path points outside of the flow snapshot?

Reply:
It can work locally.